### PR TITLE
chore(addTestingAndMocking) - Increased testing coverage and added mocks - TECH-882

### PR DESCRIPTION
**Pull request by gabedealmeida**

### CODE REVIEW

File: src/lib/ingest.ts [hunk 1]
```diff
@@ -29,7 +29,8 @@ export default async function ingest(): Promise<void> {
   logger.info(`> Got current block number: ${latestBlock}`)
 
   // get the highest block that has already been processed, so that we can start ingesting logs from that block on and not start from the beginning.
-  const highestBlock = await getHighestBlock()
+  // const highestBlock = await getHighestBlock()
```
Comment (kdsbatra) [inline src/lib/ingest.ts]: is this meant to be reverted back?

Comment (gabedealmeida) [reply src/lib/ingest.ts]: Yes

**Comment by kdsbatra** [comment]
we also need circleci to run tests as well before deploying.

File: src/utils/__tests__/generators/tokenTransfer.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    logIndex: Number(faker.random.numeric(2)),
+    blockNumber: Number(faker.random.numeric(8)),
+    ...overrides
+  }
+
+  await sequelize.query(
+    format(
+      `INSERT INTO "TokenTransfer" ("tokenType", "tokenAddress", "fromAddress", "toAddress", "operator", "tokenId", "value", "transactionHash", "logIndex", "blockNumber") VALUES %L ON CONFLICT DO NOTHING;`,
+      [value].map(Object.values)
+    )
```
Comment (kdsbatra) [inline src/utils/__tests__/generators/tokenTransfer.ts:29]: no test on checking the result of this insert?

**Pull request closed without merging**